newlib/libc/tinystdio/conv_flt.c

```diff
@@ -245,7 +245,35 @@ conv_flt (FLT_STREAM *stream, FLT_CONTEXT *context, width_t width, void *addr, u
                 return 0;
 	    }
         }
-	break;
+        if (p != pstr_nfinity + 3)
```

Comment:
looks like this will fail with `infinity(abcd)`? Perhaps add another test to check?

Reply:
testcase is added and fix is applied.